### PR TITLE
feat(as): Fetches data for municipalities

### PR DESCRIPTION
**Pull request by Rikkiabb**
# ...
https://app.asana.com/0/1201664508282204/1201645065535610

## What
Fetches municipality data from Veita in data gathering step

## Why
Needed data to process application

## Checklist:

- [X] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] Formatting passes locally with my changes
- [X] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/api/domains/municipalities-financial-aid/src/lib/municipalitiesFinancialAid.resolver.ts [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
+import { Args, Query, Resolver } from '@nestjs/graphql'
 import type { User } from '@island.is/auth-nest-tools'
 import {
   IdsUserGuard,
   ScopesGuard,
   CurrentUser,
 } from '@island.is/auth-nest-tools'
 import { UseGuards } from '@nestjs/common'
+import type { MunicipalityControllerGetByIdRequest } from '@island.is/clients/municipalities-financial-aid'
+import type { Municipality } from '@island.is/financial-aid/shared/lib'
```
Comment (MargretFinnboga) [inline libs/api/domains/municipalities-financial-aid/src/lib/municipalitiesFinancialAid.resolver.ts:10]: are these two suppose to be import type? And are they even used? 👀

**Review by MargretFinnboga — COMMENTED** [review]
minor comment and formatting is failing, are you sure this branch runs? Because after pulling on the feature branch my branch failed ❌

**Comment by Rikkiabb** [comment]
> minor comment and formatting is failing, are you sure this branch runs? Because after pulling on the feature branch my branch failed ❌

This works, what fails locally for you?

**Review by MargretFinnboga — APPROVED** [review]
👏

**Pull request merged**